gitpod-io/gitpod
# [ws-manager] Add team and project to logs and traces

## Description
Add team and project to ws-manager logs and traces. 

Logs:
![logs](https://user-images.githubusercontent.com/24721048/173405308-9a6e56fd-7f31-45de-b558-2bfc6685364e.png)

Traces:
![traces](https://user-images.githubusercontent.com/24721048/173405319-12adcf18-63d8-47da-995a-8b38ecebea58.png)


## Related Issue(s)
Fixes https://github.com/gitpod-io/gitpod/issues/9374

## How to test
- Create a team and within that team a project
- Open a workspace within the project
- Check that the pod contains project and team labels
- Search for team id/projectId in logs explorer
- Search for  team id/projectId in honeycomb

## Release Notes

```release-note
NONE
```
